lib/commons/aria/get-role.js

```diff
@@ -82,8 +82,8 @@ function getInheritedRole(vNode, explicitRoleOptions) {
   return getInheritedRole(vNode.parent, explicitRoleOptions);
 }
 
-function resolveImplicitRole(vNode, explicitRoleOptions) {
-  const implicitRole = getImplicitRole(vNode);
+function resolveImplicitRole(vNode, includeChromiumRoles, explicitRoleOptions) {
```

Comment:
Why split this into two arguments? How about:

```suggestion
function resolveImplicitRole(vNode, { includeChromiumRoles, ...explicitRoleOptions }) {
```